DAO.sol

```diff
@@ -902,16 +954,28 @@ contract DAO is DAOInterface, Token, TokenCreation {
         if (blocked[_account] == 0)
             return false;
         Proposal p = proposals[blocked[_account]];
-        if (now > p.votingDeadline) {
+        if (p.open) {
             blocked[_account] = 0;
             return false;
         } else {
             return true;
         }
     }
 
+    function isSplitBlocked(address _account) internal returns (bool) {
+        if (splitBlocked[_account] == 0)
+            return false;
+        Proposal p = proposals[splitBlocked[_account]];
+        if (p.open) {
+            splitBlocked[_account] = 0;
+            return false;
+        } else {
+            return true;
+        }
+    }
+
     function unblockMe() returns (bool) {
-        return isBlocked(msg.sender);
+        return isBlocked(msg.sender) && isSplitBlocked(msg.sender);
```

Comment:
This is something was really bothering me with the v1.0 DAO. Please as part of this PR rename `isBlocked` to `getOrModifyBlocked()`